setuptools/py36compat.py

```diff
@@ -13,10 +27,10 @@ class Distribution_parse_config_files:
     as implemented in distutils.
     """
     def parse_config_files(self, filenames=None):
-        from configparser import ConfigParser
+        from setuptools.extern.six.moves.configparser import ConfigParser
```

Comment:
The docstring of this class instructs not to change this class. It's a copy/paste from distutils and must remain that way to avoid divergence of behavior. Maybe soon setuptools will decouple itself from distutils, but at least as devised, this code is meant to be a backport of what's found in stdlib.

Reply:
OK, but it is currently not used (in master), `Distribution_parse_config_files.parse_config_files` is never called, again, see: https://github.com/pypa/setuptools/issues/889#issuecomment-318363389

Reply:
Ugh. What a mess. If setuptools needs to override the behavior found in Python, rather than adopt it, then that's what we should do... and not have a comment that's now diverged from the implementation.